src/public/js/error.js

```diff
@@ -0,0 +1 @@
+export default () => `<h1>Error</h1>`;
```

Comment:
In `src/api/orgs.js` we can respond with 3 possible error codes, `200`, `401` and `500`. If there is an `error` with sending the request to github, then this will be the page that will be shown